plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -308,7 +331,18 @@ export const MembersListCard = (props: {
         {showAggregateMembers && loadingDescendantMembers ? (
           <Progress />
         ) : (
-          memberList
+          <>
+            <TextField
```

Comment:
Does these need some padding to the members underneath? 

In the design it looks a little tight?